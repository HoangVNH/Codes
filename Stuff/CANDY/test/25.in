10
49
XXXVVVVVXXXDDDXXXXDXXXXXXXVVVVVDDDDDVVVDDDDDDVVVV
32
DDXDVVVVVVVVVVVVXXXXXXDDDDXXXXXV
35
VVVVVDDXXXDDDDVXXXXDDDDDVXXXXXXXXVV
21
DDDDDDDDDXXXVVVVVVVVD
46
DDDDDDDDDDDXXXXDDDVVVVVXDDXDDDDVVVVVXXXXXDDDDD
41
XXXDDDDVVVVDDDDVVXXXXXDDDDDXXXXVVVVVVVVDD
50
DDXXXXXXXXVVDDDDDDDDDDDDDXXXXXVVVVVDVVVVVVVDDXDDDD
45
VVVVVXXXXXVVVVVVXXXXXDDDDDXXXXXVVVVVVVXVVVVVV
42
XXXXXVVVDDXXXDDDDXXXXXXXVVXDDDXDDDDDDDVDDD
30
XXXVVVVVVDDXXXXXVXXXXXDDVVVVVX
