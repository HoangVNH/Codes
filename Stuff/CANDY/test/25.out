47
31
33
20
45
40
48
44
41
28
